lcc_lokavaluto_app_connection/models/wallet.py

```diff
@@ -126,22 +126,21 @@ def get_wallet_commission_rule(self):
     def _compute_is_reconversion_allowed(self):
         all_rules = self.env["reconversion.rule"].search([("active", "=", True)])
```

Comment:
no `order="sequence"` here ?
If there is no sequence field in reconversion rule model, I remember we said to add one, didn't we?

Reply:
my bad, corrected